src/univariate/continuous/chi.jl

```diff
@@ -80,9 +80,11 @@ end
 
 #### Evaluation
 
-logpdf(d::Chi, x::Real) = (ν = d.ν;
+function logpdf(d::Chi, x::Real)
+    x < 0 && return 0.0
+    ν = d.ν
     (1 - ν/2) * logtwo + (ν - 1) * log(x) - x^2/2 - loggamma(ν/2)
```

Comment:
One possible implementation would be e.g., similar to https://github.com/JuliaStats/StatsFuns.jl/blob/13e231a0a22e716426b73cb87ff3b8b24e33aaf1/src/distrs/gamma.jl#L19-L24 and
```suggestion
    ν, _x = promote(d.ν, x)
    xsq = _x^2
    val = (xlogy(ν - 1, xsq / 2) - xsq + logtwo) / 2 - loggamma(ν / 2)
    return x < zero(x) ? oftype(val, -Inf) : val
```